samples/web/content/apprtc/js/call.js

```diff
@@ -191,11 +192,12 @@ Call.prototype.connectToRoom_ = function(roomId, mediaPromise, turnPromise) {
     // and have media and TURN. Since we send candidates as soon as the peer
     // connection generates them we need to wait for the signaling channel to be
     // ready.
-    Promise.all([turnPromise, mediaPromise]).then(function() {
-      this.startSignaling_();
-    }.bind(this)).catch(function(error) {
-      this.onError_('Failed to start signaling: ' + error.message);
-    }.bind(this));
+    Promise.all([this.getTurnServersPromise_, this.getMediaPromise_])
```

Comment:
OK, I misunderstood the code. I still think we don't want to start the camera on the landing page until Chuck's change is in. It's strange to have the cam light on without any clear indication as to why.